.travis.yml

```diff
@@ -0,0 +1,10 @@
+# Use containers instead of full VMs for faster startup.
+sudo: false
+
+language: python
+python:
+  - "3.6"
+install:
+  - "pip install -r requirements/development.txt"
+script:
+  - "python manage.py test"
```

Comment:
I see you changed the README, does that include these changes to the setup instructions?

Reply:
The awesome thing about Travis is that you don't have to do anything. For now, it will just run on all PR's and report if the tests didn't pass at the bottom ('All checks have passed'). It's the first step in creating a continuous integration pipeline